ui/app/styles/core.scss

```diff
@@ -44,6 +44,7 @@
 
 // Helpers
 @import './helper-classes/other';
+@import './helper-classes/colors';
```

Comment:
welp, forgot to add that one to the imports. saw some good improvements in the app once I added it in :)